src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/BookingService.kt

```diff
@@ -1255,7 +1255,7 @@ class BookingService(
     booking.extensions.add(extension)
     updateBooking(booking)
 
-    if (booking.service == ServiceName.approvedPremises.value && booking.application != null && user != null) {
+    if (booking.service == ServiceName.approvedPremises.value && booking.application != null && user != null && (!manualBookingsDomainEventsDisabled || booking.placementRequest != null)) {
```

Comment:
Could we break this conditional into a private function called maybe `shouldCreateDomainEventForBooking` or similar?